src/Nri/Ui/UiIcon/V1.elm

```diff
@@ -2031,26 +2031,56 @@ graduateCap =
 apple =
-    Nri.Ui.Svg.V1.init "0 0 88 89"
+    Nri.Ui.Svg.V1.init "0 0 86 87"
         [ Svg.g
-            [ Attributes.stroke "none"
-            , Attributes.strokeWidth "1"
-            , Attributes.fill "none"
-            , Attributes.fillRule "evenodd"
+            [ Attributes.width "100%"
+            , Attributes.height "100%"
+            , Attributes.fill "currentcolor"
+            , Attributes.viewBox "0 0 86 87"
+            , Attributes.version "1.1"
             ]
             [ Svg.g
-                [ Attributes.transform "translate(-30.100000, -105.000000)"
-                , Attributes.fill "currentcolor"
-                , Attributes.fillRule "nonzero"
+                [ Attributes.stroke "none"
+                , Attributes.strokeWidth "1"
+                , Attributes.fill "none"
+                , Attributes.fillRule "evenodd"
                 ]
-                [ Svg.g
-                    [ Attributes.transform "translate(30.972117, 105.800000)"
+                [ Svg.path
+                    [ Attributes.d "M49.4192778,84.2160267 C47.3020778,83.3332167 45.2903778,82.4699267 43.2278778,82.5324267 C41.3020778,82.5597707 39.3372778,83.1457067 37.2551778,83.7629267 C33.2824778,84.9426267 28.7746778,86.2785267 23.2821778,84.1183967 C16.2704778,81.3605967 13.0121778,78.1105967 9.45817778,73.0713967 C-2.15482222,56.6303967 -2.53782222,31.6533967 8.66520778,21.6843967 C13.8332078,17.0827967 20.7592078,15.0945967 27.6572078,16.2273967 C31.9658078,16.9344267 37.5947078,18.2156967 41.9812078,20.4500967 C41.4499578,17.6297967 39.9656078,14.9500967 38.7390078,13.8641967 C33.8562078,9.5516967 30.8328078,8.5985967 28.8328078,7.9735967 L28.0359278,7.7157867 C27.1179578,7.4110967 26.6218278,6.4188867 26.9304278,5.5009867 C27.2351178,4.5830167 28.2312278,4.0907867 29.1452278,4.3954867 L29.8835078,4.6337667 C32.0280078,5.3056467 35.6179078,6.4345667 41.0595078,11.2431667 C41.8720078,11.9619167 42.6767078,13.0048667 43.3837078,14.2431667 C44.3954078,11.4306667 46.1493078,9.0400667 49.0712078,5.9892667 C53.9228078,0.930666704 62.1732078,0.043966704 62.5202078,0.00876670401 C63.0709878,-0.045921296 63.6100078,0.157206704 63.9811078,0.563456704 C64.3521978,0.965796704 64.5162678,1.5243967 64.4186078,2.0634567 C64.3326698,2.5400167 62.7663078,10.6493567 55.8522078,17.7114567 C58.2702078,17.1255167 60.6803078,16.7036567 62.8170078,16.6020567 C67.8990078,16.3559667 73.3090078,18.1879567 77.6530078,21.6176567 C88.4690078,30.1606567 87.9620078,57.0906567 76.6764478,73.0706567 C72.5866478,78.8636567 70.0045478,82.0472567 60.4034478,85.5086567 C59.1417478,85.9617767 57.9307478,86.1496267 56.7745478,86.1496267 C54.0677778,86.1496267 51.6380778,85.1379267 49.4192778,84.2160267 Z M51.5945778,8.4112267 C47.3211778,12.8643267 46.0203778,15.3526267 45.7195778,20.8292267 C55.0047778,16.1026267 58.8405778,8.0632267 60.2505778,3.9812267 C57.7585778,4.5830867 54.0437778,5.8604267 51.5945778,8.4112267 Z"
+                    , Attributes.fill "#707070"
```

Comment:
This is setting the fill color to gray, which I don't think is what's actually desired. In the customizable view, for instance, the apple should turn green when I select green:

<img width="656" alt="image" src="https://user-images.githubusercontent.com/8811312/234418224-f164dbcb-2d9d-4fd9-aa45-cffb333fcc47.png">

Also if you hover over the icons, their color is supposed to change, and the apple's color doesn't:

<img width="270" alt="image" src="https://user-images.githubusercontent.com/8811312/234418345-e9410cba-2d99-4374-bad1-7405ed65bb24.png">

Reply:
Oh jeez, it took me embarrassingly long to realize that customization view was just a scroll down away 😅. But I think I got it now, setting the fill to `"currentcolor"` as the others gets the desired behaviors.